specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,861 @@
+            "in": "path",
+            "description": "Name of the Container Apps Job.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "OK",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Terminate a Container Apps Job": {
+            "$ref": "./examples/ContainerAppsJob_Terminate.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerAppsJobs/{containerAppsJobName}/listSecrets": {
+      "post": {
+        "tags": [
+          "ContainerAppsJobs"
+        ],
+        "summary": "List secrets for a container apps job",
+        "operationId": "ContainerAppsJobs_ListSecrets",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "containerAppsJobName",
+            "in": "path",
+            "description": "Name of the Container Apps Job.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/JobSecretsCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Container Apps Job Secrets": {
+            "$ref": "./examples/ContainerAppsJob_ListSecrets.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "JobConfiguration": {
+      "description": "Non versioned Container Apps Job configuration properties",
+      "type": "object",
+      "required": [
+        "replicaTimeout",
+        "triggerType"
+      ],
+      "properties": {
+        "secrets": {
```

Comment:
>secrets [](http://example.com/codeflow?start=9&length=7)

Your backend needs to null or empty out this property in all GET, PUT, and PATCH payloads.

Reply:
@mentat9 we will handle this in the backend logic.